src/MongoDB/Logging/functions.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2016-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "bson/bson.h"
+#include "mongoc/mongoc.h"
+
+#include <php.h>
+
+#include "php_phongo.h"
+#include "phongo_error.h"
+#include "phongo_log.h"
+
+#define PHONGO_LOG_DOMAIN "php"
```

Comment:
Note: if the `log()` function is relocated to PHPLIB this can be removed. See https://github.com/mongodb/mongo-php-driver/pull/1395#discussion_r1324787456 for more context.

Reply:
This was removed because the `mongoc_log()` internal function takes a `domain` variable.